cli/envfile.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// reads a file of line terminated key=value pairs, and overrides any keys
+// present in the file with additional pairs specified in the override parameter
+func readKVStrings(files []string, override []string) ([]string, error) {
+	envVariables := []string{}
+	for _, ef := range files {
+		parsedVars, err := ParseEnvFile(ef)
+		if err != nil {
+			return nil, err
+		}
+		envVariables = append(envVariables, parsedVars...)
+	}
+	// parse the '-e' and '--env' after, to allow override
+	envVariables = append(envVariables, override...)
+
+	return envVariables, nil
+}
+
+// ParseEnvFile reads a file with environment variables enumerated by lines
+func ParseEnvFile(filename string) ([]string, error) {
```

Comment:
Is it necessary to set upper case of ParseEnvFile?

Reply:
I ignore this problem. I'll modify it and thanks for your suggestion!

Reply:
In the test file, i call the ParseEnvfile function, so i change it back.